benchmark/common.js

```diff
@@ -44,7 +44,7 @@ Benchmark.prototype._parseArgs = function(argv, configs) {
   for (const arg of argv) {
     const match = arg.match(/^(.+?)=([\s\S]*)$/);
     if (!match || !match[1]) {
-      console.error('bad argument: ' + arg);
+      console.error(`bad argument: ${arg}`);
```

Comment:
I think this would imply we need `inspect()` for `arg` (as we do for Error objects), while `arg` is a simple string.